### PR TITLE
Add request DTOs

### PR DESCRIPTION
**Pull request by janinko**

### CODE REVIEW

File: dto/src/main/java/org/jboss/pnc/dto/requests/BuildPushRequest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+/**
+ *
+ * @author Honza Brázdil &lt;jbrazdil@redhat.com&gt;
+ */
+@Data
+@Builder(builderClassName = "Builder")
+public class BuildPushRequest {
+
+    private String tagPrefix;
```
Comment (jbartece) [inline dto/src/main/java/org/jboss/pnc/dto/requests/BuildPushRequest.java]: It should be final. 
```suggestion
    private final String tagPrefix;
```

File: dto/src/main/java/org/jboss/pnc/dto/requests/BuildPushRequest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    @NotNull
+    private Integer buildId;
```
Comment (jbartece) [inline dto/src/main/java/org/jboss/pnc/dto/requests/BuildPushRequest.java]: It should be final.
```suggestion
    private final Integer buildId;
```

File: dto/src/main/java/org/jboss/pnc/dto/requests/BuildPushRequest.java (hunk 1 above)
Comment (janinko) [inline dto/src/main/java/org/jboss/pnc/dto/requests/BuildPushRequest.java]: final

File: dto/src/main/java/org/jboss/pnc/dto/requests/GroupBuildPushRequest.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+/**
+ *
+ * @author Honza Brázdil &lt;jbrazdil@redhat.com&gt;
+ */
+@Data
+@Builder(builderClassName = "Builder")
+public class GroupBuildPushRequest {
+
+    private String tagPrefix;
```
Comment (jbartece) [inline dto/src/main/java/org/jboss/pnc/dto/requests/GroupBuildPushRequest.java]: Final
```suggestion
    private final String tagPrefix;
```

File: dto/src/main/java/org/jboss/pnc/dto/requests/GroupBuildPushRequest.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    @NotNull
+    private Integer groupBuildId;
```
Comment (jbartece) [inline dto/src/main/java/org/jboss/pnc/dto/requests/GroupBuildPushRequest.java]: final
```suggestion
    private final Integer groupBuildId;
```

**Comment by janinko** [comment]
Do not merge, I'll squash it before merging.

**Comment by janinko** [comment]
retest this please

**Comment by janinko** [comment]
retest this please

**Comment by janinko** [comment]
retest this please

1 similar comment

**Pull request merged**